MAUI/MauiAppB2C/MainPage.xaml.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using MauiB2C.MSALClient;
+using Microsoft.Identity.Client;
+
+namespace MauiB2C;
+
+public partial class MainPage : ContentPage
+{
+	public MainPage()
+	{
+		InitializeComponent();
+	}
+
+    private async void OnSignInClicked(object sender, EventArgs e)
+    {
+        try
+        {
+            // attempt silent login.
+            // If this is very first time or user has signed out, it will throw MsalUiRequiredException
+            AuthenticationResult result = await PCAWrapperB2C.Instance.AcquireTokenSilentAsync(B2CConstants.Scopes).ConfigureAwait(false);
+
+            // show the IdToken
+            await ShowMessage("AcquireTokenTokenSilent call IdToken", result.IdToken).ConfigureAwait(false);

```

Comment:
Please do not display the id token. Can you iteratate through the ClaimsPrincipal instead and print out all the claims?